src/main.ts

```diff
@@ -250,6 +255,7 @@ async function getTsServerRepoResult(
 
         console.log(`Issue found in ${newTsServerPath} (new):`);
         console.log(insetLines(prettyPrintServerHarnessOutput(newSpawnResult.stdout, /*filter*/ false)));
+        await fs.promises.writeFile(rawErrorsScriptPath, prettyPrintServerHarnessOutput(newSpawnResult.stdout, /*filter*/ false));
```

Comment:
Can you please confirm that this is supposed to end up in `rawErrorsScriptPath` and not `path.join(resultDirPath, rawErrorsScriptFileName)` (i.e. where used to copy it to below)?

Reply:
I ran the tests and raw error file is not generated if it ends up at `rawErrorsScriptPath`. So I changed it back to end up at where it was copied over (`path.join(resultDirPath, rawErrorsScriptFileName)` ). It required parameters like `resultDirPath` that have later been defined in file, so I cannot make the file write to the final destination directly here.

Reply:
Sorry, the diff made it hard to tell that the copy was in the caller.  This is fine.